core/che-core-api-dto/src/test/java/org/eclipse/che/dto/definitions/DTOHierarchy.java

```diff
@@ -60,4 +62,6 @@
         void setShadowedField(GrandchildDto v);
     }
 
+    public interface GrandchildWithoutDto extends Child, Closeable {
```

Comment:
Closeable is just additional interface for testing propose. I am ready to produce second child if you  think it's less confusing :-)